app/controllers/debt_test_controller.rb

```diff
@@ -7,6 +7,10 @@ def parent_template
     end
   end
 
+  def exclude_syndicated_iframe_resizer?
+    return true if syndicated_tool_request?
```

Comment:
just thought that `syndicated_tool_request?` will work don't need the `return true` bit